keyboards/kyria/keymaps/mattir2/keymap.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020 Matthew Gilbert <hello+github@mattir.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers {
+  QWERTY = 0,
```

Comment:
This doesn't need to be set, actually, as the enum will start at 0, automatically. 
```suggestion
  QWERTY,
```